index.html

```diff
@@ -23,6 +23,8 @@ <h1 class="is-size-1">Perfect Ambient Generator</h1>
         <button class="button is-info" id="searchButton" type="button">Submit</button>
     </div>
 
+    <div id="bookResults"></div>
```

Comment:
Can you update your branch and merge the latest main branch into your feature/branch because I believe you're missing some changes from main.

Reply:
Sure, just updated my branch.